GitUI/UserControls/FileStatusList.cs

```diff
@@ -1021,10 +1021,19 @@ private void UpdateColumnWidth()
             if (!_updatingColumnWidth)
             {
                 _updatingColumnWidth = true;
-                columnHeader.Width = GetWidth();
+
+                int width = GetWidth();
+                if (columnHeader.Width != width)
+                {
+                    columnHeader.Width = width;
+                    FileStatusListView.ForceRepaint();
```

Comment:
Could we use something like (pseudocode)
```
FileStatusListView.BeginInvoke(() => FileStatusListView.Invalidate());
```
This should invalidate the control asynchronously.

Reply:
Works indeed. Though the compiler wants a `new Action()` around the lambda.